### DISCUSSION TITLE
Allow support for more specific bitrates and monaural audio

### DISCUSSION THREAD

**Discussion by ryanginn** [AzuraCast/AzuraCast#6847] [Ideas]
In my opinion, I think support for more specific bitrates and support for mono streams would be a great benefit, for example - being able to manually be able to input what bitrate for which mountpoint, for example 80kbps.

Many thanks.

**Comment by BusterNeece** [comment]
@ryangontv You should submit this over at https://features.azuracast.com/, where we manage feature requests and the community can vote on which ones they'd like to see prioritized.
